tests/sentry/integrations/github/test_integration.py

```diff
@@ -608,9 +608,8 @@ def test_get_trees_for_org(self):
             self._caplog.set_level(logging.INFO, logger="sentry")
             # Check that the cache is clear
             assert cache.get("githubtrees:repositories:foo:Test-Organization") is None
-            assert cache.get("githubtrees:repo:Test-Organization/foo") is None
+            assert cache.get("github:repo:Test-Organization/foo:source-code") is None
```

Comment:
The key is changed in here:
<img width="520" alt="image" src="https://user-images.githubusercontent.com/44410/204310695-cdcf3d6e-dc4f-4bed-ade9-3486802d3a20.png">